src/utilities/flex/TransientTokenGenerator.js

```diff
@@ -0,0 +1,19 @@
+
+'use strict';
+
+function TransientTokenGenerator(jwt)
```

Comment:
The name should **NOT** be `TransientTokenGenerator`. Change it to `TransientTokenUtility`.